Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -3394,6 +3402,67 @@ def get_modified_remote_data_command(client: Client, params: Dict[str, str],
     return GetModifiedRemoteDataResponse(list(new_modified_records_ids))
 
 
+def qradar_get_events_polling_command(client: Client,
+                                      params,
+                                      args,
+                                      ) -> CommandResults:  # pragma: no-cover (tested in test-playbook)
+    """A polling command to get events from QRadar offense
+
+    Args:
+        client (Client): The QRadar client to use.
+        params (dict): Parameters passed to the command.
+        args (dict): Demisto arguments.
+
+    Raises:
+        DemistoException: If the search failed.
+
+    Returns:
+        CommandResults: The results of the command.
+    """
+    ScheduledCommand.raise_error_if_not_supported()
+    interval_in_secs = int(args.get('interval_in_seconds', 60))
+    offense_id = args.get('offense_id')
+    search_id = args.get('search_id')
+    if not search_id:
+        search_id = create_events_search(client,
+                                         params.get('fetch_mode'),
+                                         params.get('events_columns'),
+                                         params.get('events_limit'),
+                                         offense_id)
+        if search_id == QueryStatus.ERROR.value:
+            raise DemistoException(f'Failed to create search. for Offense {offense_id}')
+        polling_args = {
+            'search_id': search_id,
+            'interval_in_seconds': interval_in_secs,
+            'polling': True,
```

Comment:
Is it possible to run this command without `polling`? If not please remove the arg.